README.md

```diff
@@ -5,14 +5,13 @@ Web service which collects and serves bug reports.
 To run it, do:
 
 ```
-BUGS_USER=<user> BUGS_PASS=<password> go run src/github.com/matrix-org/rageshake/main.go PORT
+go get github.com/constabulary/gb/...
+gb build
```

Comment:
well, I'm about to add a dep on https://godoc.org/github.com/google/go-github, which changes that.